src/lang/std/sketch.ts

```diff
@@ -849,8 +850,8 @@ export const tangentialArcTo: SketchLineHelper = {
       pathToNode,
```

Comment:
Above and below here are AST mod stuff specific to `tangentialArcTo`, so I suspect we need to change more stuff in this block.